crates/router/src/consts.rs

```diff
@@ -153,6 +153,9 @@ pub const DEFAULT_ALLOWED_DOMAINS: Option<HashSet<String>> = None;
 /// Default hide card nickname field
 pub const DEFAULT_HIDE_CARD_NICKNAME_FIELD: bool = false;
 
+/// Show card form by default for payment links
+pub const DEFAULT_SHOW_CARD_FORM: bool = true;
```

Comment:
This is true, where as in api_models default is 'false'!

Reply:
yes my bad, made the default value 'true' everywhere now